src/main/java/org/alliancegenome/curation_api/interfaces/document/GeneExpressionDocumentInterface.java

```diff
@@ -0,0 +1,30 @@
+package org.alliancegenome.curation_api.interfaces.document;
+import java.util.HashMap;
+
+import org.alliancegenome.curation_api.model.document.es.GeneExpressionDocument;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.curation_api.view.View;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import com.fasterxml.jackson.annotation.JsonView;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/genexpression/document")
```

Comment:
I would call this
/gene-expression
and leave out /document as that is added already as /expressions-docuements